src/templates/pad.html

```diff
@@ -464,7 +464,7 @@ <h1>Skin Builder</h1>
       // @license-end
   </script>
 
-  <script type="text/javascript" src="../socket.io/socket.io.js?v=<%=settings.randomVersionString%>"></script>
+  <script type="text/javascript" src="../socket.io/socket.io.js"></script>
```

Comment:
This should not be here :)

Reply:
Unfortunately I had to remove the random version tag by force, otherwise the socket.io client script would not load!